cmake/cmake_lists.jinja

```diff
@@ -0,0 +1,73 @@
+############################################################################
```

Comment:
Note: project already uses EmPy (in `genmsg`), maybe better use that template engine?